sql-statements/sql-statement-admin-cleanup.md

```diff
@@ -0,0 +1,74 @@
+---
+title: ADMIN CLEANUP 
```

Comment:
Thanks for providing the input. @shichun-0415 will move the content of this PR to https://github.com/pingcap/docs-cn/pull/12952.